goldmane/proto/api.proto

```diff
@@ -334,6 +334,15 @@ message FlowKey {
   // name and namespace fields.
   EndpointType source_type = 3;
 
+  // SourceIP is the IP address of the source endpoint.
+  string source_ip = 16;
+
+  // SourcePort is the source port number.
+  int64 source_port = 17;
```

Comment:
As mentioned above, please remove source ports for now; let's get IPs working and then worry about ports if we get a compelling use case